addons/playersuppression/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\ace\addons\blank
```

Comment:
playersuppression here.
required for building the pbo while the game is running or something